README.md

```diff
@@ -73,11 +73,12 @@ Refer to the [`podman push`](http://docs.podman.io/en/latest/markdown/podman-man
 
 ## Action Outputs
 
-`registry-path`: The registry path to which the image was pushed.<br>
-For example, `quay.io/username/spring-image:v1`.
+`registry-paths`: The List of registry paths to which the tag(s) were pushed.<br>
+For example, `quay.io/username/spring-image:v1,quay.io/username/spring-image:v2`.
```

Comment:
Is this what the output actually looks like? It should be a JSON-parsable array like: 

`[ "quay.io/username/spring-image:v1", "quay.io/username/spring-image:v2" ]`

You may want to use `JSON.stringify(the object/array)` when setting the output.

Reply:
And make clear that this is a JSON array.